lib/vanagon/utilities.rb

```diff
@@ -281,5 +281,16 @@ def erb_file(erbfile, outfile = nil, remove_orig = false, opts = { :binding => b
       FileUtils.rm_rf erbfile if remove_orig
       outfile
     end
+
+    # Final
+    # - 1.0.0
+    # - 3.0.5.1
+    # - 4.3.0-5
+    # Nonfinal
+    # - 1.0.0-34-g1sadjf9283
+    # - 4.2.1.2344.gfdqw83rh
+    def final_version?(version)
+      !version.match(/g[a-f0-9]{7}$/)
```

Comment:
I suppose if you wanted a different check, you could say, is this versioned from a git tag or is the version hard-coded? If it's versioned from a tag, is the build we're doing current on a tag or not?